.dockerignore

```diff
@@ -0,0 +1,4 @@
+# More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
```

Comment:
this dockerignore file can be removed since we only copy dedicated folders into the container.